src/tests/tracing/eventpipe/pauseonstart/pauseonstart.cs

```diff
@@ -28,7 +28,7 @@ public static async Task<bool> TEST_RuntimeResumesExecutionWithCommand()
             var server = new ReverseServer(serverName);
             Task<bool> subprocessTask = Utils.RunSubprocess(
                 currentAssembly: Assembly.GetExecutingAssembly(),
-                environment: new Dictionary<string,string> { { Utils.DiagnosticsMonitorAddressEnvKey, serverName } },
+                environment: new Dictionary<string,string> { { Utils.DiagnosticPortsEnvKey, $"{serverName},connect,suspend" } },
```

Comment:
connect and suspend are the defaults and I'd guess our simple use-case tutorial won't include them. I'd recommend making the test mirror as closely as possible what we expect users to do which hopefully shouldn't make any difference here.